Automattic/pocket-casts-android
# Update now playing chip on watch

## Description
This updates the UI of the Now Playing chip.

The current designs only specify that this chip should be showing when an episode is playing, but I'm always presenting it currently because I think otherwise it's too difficult to start playing an episode when you first open the app (I certainly think we need it until I get the pager added).

I'm also using an ellipsis when the text fields are too long instead of clipping them because clipping would cut off the field at the end of the nearest word and, depending on the length and positions of the words, this could result in some absurdly short text on the chip (i.e., "This is spectacular" could get cut so that it would show only "This is" if clipping was used).

## Testing Instructions
1. Open a fresh install of the app
2. Observe the now playing chip has a play icon and says "Now Playing"
3. Log into an account and wait for a bit for it to load (it's pretty slow because it's still doing this over Bluetooth... this is something on my todo list)
4. Once the account has loaded with the currently queued episode, observe the Now Playing chip has a play icon, the podcast artwork as the background, along with the podcast title and episode title.
5. Tap on the Now Playing chip
6. Start playing the podcast
7. Swipe back
8. Observe that the Now Playing chip now says "Now Playing" in place of the podcast name and the play icon has been replaced with an animated equilizer.

## Screenshots or Screencast 

https://user-images.githubusercontent.com/4656348/230661981-4d798cf3-b2e4-44f3-9fca-e55da5ea4e9b.mov

## Checklist
